src/redux/books/books.js

```diff
@@ -1,43 +1,37 @@
 // App Actions to update state
 const ACTIONS = {
   ADDED_BOOK: 'bookStore/books/ADDED_BOOK',
-  REMOVED_BOOK: 'bookStore/books/REMOVED_BOOK'
-}
+  REMOVED_BOOK: 'bookStore/books/REMOVED_BOOK',
+};
 
 // an object contains Different types of action as
 // it's key and the value is an specific function
 const cases = {
-  [ACTIONS.ADDED_BOOK]: (state, action) => {
-    return [...state, action.payload]
-  },
+  [ACTIONS.ADDED_BOOK]: (state, action) => [...state, action.payload],
   [ACTIONS.REMOVED_BOOK]: (state, action) => {
-    let newState = [...state]
-    newState = newState.splice(action.index, 1)
-    return [...newState]
-  }
-}
+    let newState = [...state];
+    newState = newState.splice(action.index, 1);
+    return [...newState];
+  },
+};
```

Comment:
- Hey, please, kindly improve on the following:

   - Your default book gets deleted correctly but if I delete them and add like 4 or 2 books, on deleting one, all the other books get deleted. Please test to see it in action so you can improve on this. :pray: